packages/@okta/vuepress-site/docs/guides/configure-access-policy/overview/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+---
+
+## What is an access policy?
+
+Access policies help you secure your APIs by defining different access and refresh token lifetimes for a given combination of grant type, user, and scope. You create policy rules that allow code to evaluate whether an application should be permitted to access specific information from your protected APIs and for how long. Access policies are specific to a particular authorization server and the client applications that you designate for the policy.
```

Comment:
"that allow code to evaluate whether" in "You create policy rules that allow code to evaluate whether an application should be permitted to access specific information from your protected APIs and for how long." above is unclear

Perhaps change to something like:

> You create policy rules to determine if an application should be permitted to access specific information from your protected APIs and for how long.